wpeverest/everest-forms
# Cache the form and entries results

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://docs.google.com/document/d/1rYjcAfrufgDEcoDmEoZZ_01fVDmmg9tcKiPkCAsmiPY/edit

### How to test the changes in this Pull Request:

1. Visit the Entries/forms page and others.
2. Check if the Query monitor doesn't show duplicate queries.
3. If QM notices are minimized this PR has performed the job required.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Cache the form and entries results for better performance.
